src/components/dev-hub/global-nav.js

```diff
@@ -30,14 +33,16 @@ const NavContent = styled('div')`
 `;
 
 const NavLink = styled(Link)`
+    font-size: ${fontSize.small};
     font-weight: 300;
     letter-spacing: 1px;
-    line-height: ${lineHeight.xxlarge};
-    padding: ${size.small} ${size.xlarge};
+    line-height: ${lineHeight.small};
+    padding: ${LINK_VERTICAL_PADDING} ${size.xlarge};
     text-decoration: none;
     &:hover,
     &[aria-current='page'] {
-        background-color: ${colorMap.greyDarkTwo};
+        /* 66 is 40% opacity in hex */
+        background-color: ${`${colorMap.greyDarkTwo}66`};
```

Comment:
we're not using hexa anywhere else, might be better to just get the resultant hex value.